addons/project/models/project.py

```diff
@@ -1630,6 +1630,11 @@ def create(self, vals_list):
                 vals['date_last_stage_update'] = fields.Datetime.now()
             # recurrence
             rec_fields = vals.keys() & self._get_recurrence_fields()
+            # import tasks with recurrence
+            if not vals.get('recurrence_id') and not rec_fields and vals.get('recurring_task') is True:
+                default_val = self.default_get(self._get_recurrence_fields())
+                vals.update(**default_val)
+                rec_fields = vals.keys() & self._get_recurrence_fields()
```

Comment:
I think the import calls the `create` method via the `_load_records_create` method. So, we could perhaps do this in that method before calling the parent method for instance. 🤔 

Moreover, don't we need to check if the feature is enabled before doing this? 🤔

Reply:
Hello @xavierbol I do the _load_records_create method. 

`don't we need to check if the feature is enabled before doing this? ` 
Case- In project configuration, Recurring is false initially. User import records with recurring field= true. When the user realized that Recurrent is false, then makes it true. in the task imported before, recurring does not happen.  I agree that this use case is not very common. Accidents happen, users mess up. 
cc @aju-odoo  @sar-odoo

Reply:
For me, if the feature is disabled then we don't must create recurrent tasks when the user imports data. Let's see with @aju-odoo.